js/whitebit.js

```diff
@@ -267,36 +267,31 @@ module.exports = class whitebit extends Exchange {
     }
 
     async fetchCurrencies (params = {}) {
-        const response = await this.v2PublicGetAssets (params);
-        //
-        //     {
-        //         "success":true,
-        //         "message":"",
-        //         "result":{
-        //             "BTC":{
-        //                 "id":"4f37bc79-f612-4a63-9a81-d37f7f9ff622",
-        //                 "lastUpdateTimestamp":"2019-10-12T04:40:05.000Z",
-        //                 "name":"Bitcoin",
-        //                 "canWithdraw":true,
-        //                 "canDeposit":true,
-        //                 "minWithdrawal":"0.001",
-        //                 "maxWithdrawal":"0",
-        //                 "makerFee":"0.1",
-        //                 "takerFee":"0.1"
-        //             }
-        //         }
-        //     }
+        const response = await this.v4PublicGetAssets (params);
+        // }
+        // "BTC": {
+        //     "name": "Bitcoin",
+        //     "unified_cryptoasset_id": 1,
+        //     "can_withdraw": true,
+        //     "can_deposit": true,
+        //     "min_withdraw": "0.001",
+        //     "max_withdraw": "2",
+        //     "maker_fee": "0.1",
+        //     "taker_fee": "0.1",
+        //     "min_deposit": "0.0001",
+        //     "max_deposit": "0",
+        //  },
+        // }
```

Comment:
I'd really appreciate, if you could follow the comment formatting precisely, compare formatting, indentation, whitespaces on the left to formatting on the right:

<img width="1627" alt="Screenshot 2021-12-24 at 13 19 12" src="https://user-images.githubusercontent.com/1294454/147344383-95be16bc-6318-4786-b91f-feb642f49260.png">

Reply:
@kroitor  sorry, I'll fix it